wiki/faq.md

```diff
@@ -233,9 +233,25 @@ only.
 ---
 ### What happens if I stake on a fork?
 
-When you stake on a fork, the stake you see is not actually legitimate. When your wallet figures out that it is on a fork, you will see these rewards disappear as the wallet corrects itself. This process happens automatically. While frustrating, once off of the fork, you are still able to stake again. 
+When you stake on a [fork](forks "wikilink"), the stake you see is not actually legitimate. When your wallet figures out that it is on a fork, you will see these rewards disappear as the wallet corrects itself. This process happens automatically. While frustrating, once off of the fork, you are still able to stake again. 
 
 ---
+
+### My pending rewards are stuck at 16384 GRC
+
+If you see your pending rewards at 16384 GRC, you have hit the maximum amount
+Gridcoin that can be sent per stake. When you stake you will only earn 16384 GRC
+and any amount of extra work done will not be included. If you keep hitting this
+limit, make sure you are running your wallet to stake 24/7 (or close to it). If you are, 
+you should get more Gridcoin to stake more frequently. 
+
+If you want to get an estimate of how likely you are to stake *before* that limit, you can try solving 
+for probability using:
+`16384/(magnitude/4) = -ln( 1 - Probability ) * 10000 / Balance * Difficulty`

```

Comment:
perhaps a note should be added here about what the probability variable _means_

Reply:
It's how likely you are to stake before that limit, so not sure how to define it without repeating myself. Was aiming to use that previous

Reply:
Maybe just "for *that* probability" would work?

Reply:
I mean that it is a confidence interval.  I would guess most people don't have a concrete understanding of those.

Reply:
Put a little sentence after it that explains it a little bit